$OpenBSD: patch-cmake_FindUmfpack_cmake,v 1.1.1.1 2014/07/10 08:59:52 zhuk Exp $
1. Allow to find UMFPACK headers.
2. Make sure AMD (math/amd) headers location is also picked up.
--- cmake/FindUmfpack.cmake.orig	Wed Feb 26 19:35:18 2014
+++ cmake/FindUmfpack.cmake	Mon Jun 30 12:13:02 2014
@@ -5,7 +5,7 @@ if (UMFPACK_INCLUDES AND UMFPACK_LIBRARIES)
   set(UMFPACK_FIND_QUIETLY TRUE)
 endif (UMFPACK_INCLUDES AND UMFPACK_LIBRARIES)
 
-find_path(UMFPACK_INCLUDES
+find_path(UMFPACK_INCLUDE_DIR
   NAMES
   umfpack.h
   PATHS
@@ -14,7 +14,17 @@ find_path(UMFPACK_INCLUDES
   PATH_SUFFIXES
   suitesparse
   ufsparse
+  umfpack
 )
+
+find_path(AMD_INCLUDE_DIR
+  NAMES
+  amd/amd.h
+)
+
+if(UMFPACK_INCLUDE_DIR)
+  set(UMFPACK_INCLUDES ${UMFPACK_INCLUDE_DIR} ${AMD_INCLUDE_DIR})
+endif()
 
 find_library(UMFPACK_LIBRARIES umfpack PATHS $ENV{UMFPACKDIR} ${LIB_INSTALL_DIR})
 
